lib/job.ml

```diff
@@ -61,7 +61,7 @@ let jobs_dir = lazy (Disk_store.state_dir "job")
 let log_path job_id =
   let open Astring in
   let jobs_dir = Lazy.force jobs_dir in
-  match String.cuts ~sep:"/" job_id with
+  match String.cuts ~sep:Fpath.dir_sep job_id with
```

Comment:
This doesn't seem right. The syntax of job IDs shouldn't depend on the platform. Probably `id_of_path` should be changed so they always use `/`.

Reply:
Changed to use `Fpath.segs` then re-concatenating the results with `/`.